src/assets/i18n/en.json5

```diff
@@ -4839,4 +4839,8 @@
   "home.recent-submissions.head": "Recent Submissions",
 
   "listable-notification-object.default-message": "This object couldn't be retrieved",
+
+  "search.filters.filter.show-tree": "Browse {{ name }} tree",
```

Comment:
Can we move this key so that it's next to the other `search.filters.filter.*` i18n keys in this same file (around line 3641)?  It's best to keep related keys together as it makes it easier for translators.

Reply:
Resolved here https://github.com/DSpace/dspace-angular/commit/e0cadec2c21b44587945edd08e17c3fda1532ed0